drivers/sensor/stm32/temp_stm32f401x.c

```diff
@@ -195,10 +195,9 @@ static int temp_stm32f401x_init(struct device *dev)
 	 *   mutually exclusive, and VBAT has precedence).
 	 */
 	tmp = adc_common->CCR;
-	tmp |= (ADC_CCR_TSVREFE << ADC_CCR_TSVREFE_Pos);
-	tmp &= ~(ADC_CCR_ADCPRE << ADC_CCR_ADCPRE_Pos);
-	tmp |= (ADC_PRESCALER_PCLK_DIV_8 << ADC_CCR_ADCPRE_Pos);
-	tmp &= ~(ADC_CCR_VBATE << ADC_CCR_VBATE_Pos);
+	tmp &= ~(ADC_CCR_VBATE | ADC_CCR_ADCPRE);
+	tmp |= ((ADC_PRESCALER_PCLK_DIV_8 << ADC_CCR_ADCPRE_Pos) |
```

Comment:
Yeah, that one is defined up earlier in the file.

The way it works is that stm32f401xe.h provides:

```
#define ADC_REG_FIELD_Msk 0xvalue
#define ADC_REG_FIELD_Pos bit_shift_value
#define ADC_REG_FIELD (ADC_REG_FIELD_Msk << ADC_REG_FIELD_Pos)
```
So for encoded values that I define myself in this file, I have to shift them.